### ISSUE TITLE
Using nextTab and previousTab on Arc Browser

### ISSUE THREAD

**Issue by asgeirtj** [philc/vimium#4465]
Hey there

I'm using Arc browser on macOS which uses a different tab system than chrome, where at least when using the tabs which are pinned on top don't work like you'd expect with nextTab previousTab. I'm looking to find a solution so I could use my bindings for these functions like in chrome.

1 Is it possible to map a key to output meta+alt+down/up? (which is default for jumping to next and prev tab in the siderbar in arc)
2 It's possible to to this with keyboard maestro/better touch tool but it takes like half a sec for the script to execute so I can't be bothered with that.

Any ideas for my problem? :D

Thanks!

**Comment by philc** [comment]
I haven't used Arc and so have no insight about its tabs, but for your second question, I use [Karabiner Elements](https://karabiner-elements.pqrs.org/) for key remapping cases like this. There's no latency.

**Comment by asgeirtj** [comment]
> I haven't used Arc and so have no insight about its tabs, but for your second question, I use [Karabiner Elements](https://karabiner-elements.pqrs.org/) for key remapping cases like this. There's no latency.

Can you check for if in an inputbox in karabiner?

**Comment by yochem** [comment]
No, I don't think you can. It is also not possible to remap keymaps like `J` in Arc settings, seems like the shortcut would need a modifier key.

**Comment by yochem** [comment]
I think currently previousTab and nextTab cycle between your last visited tabs

**Comment by jakepearson** [comment]
Ideas on how to make `previousTab`/`nextTab` just move to the next tab instead of navigating history? Alternatively, is there a way in vimium to have a key trigger a shortcut. For example, in Arc I have next tab bound to `alt+d` but would love that be just `d` if no text box is active.
